benchmark/bench-report/BenchReport.hs

```diff
@@ -224,10 +224,7 @@ selectBench Options{..} f =
     cutOffMultiples = 1 + cutOffPercent / 100
 
     filterPred x =
-        case benchType of
-            Just (Compare _) ->
-                x <= negate cutOffPercent || x >= cutOffPercent
-            _ -> True
+        x <= negate cutOffPercent || x >= cutOffPercent
```

Comment:
If we are not comparing, then how does cutOffPercent make sense? I am missing something.

Reply:
Ok, I guess Compare is for comparison groups, the diff applies in the other case as well. I have not looked at the code.